src/app/Plans/components/PlanActionsDropdown.tsx

```diff
@@ -103,14 +103,9 @@ const PlansActionsDropdown: React.FunctionComponent<IPlansActionDropdownProps> =
         toggleOpen={toggleDeleteModal}
         mutateFn={() => deletePlanMutation.mutate(plan)}
         mutateResult={deletePlanMutation}
-        title="Delete migration plan"
+        title="Delete migration plan?"
```

Comment:
I'd rewrite this headline to emphasize that deleting this migration plan permanently impacts users' data, and rephrase the body so that we avoid repeating the same language from the headline:

**Permanently delete migration plan?**
All data for migration plan "name" will be lost.